dom_test.py

```diff
@@ -0,0 +1,199 @@
+import csv
+import difflib
+import os
+import re
+
+from lxml import etree
+
+ignoredAttrib = {'style', 'type'}
+matched21 = {}
+matched12 = {}
+nodes_info = {1: {}, 2: {}}
+chrome_tree = None
+firefox_tree = None
+THRESHOLD_LEVEL = 0.75
+THRESHOLD_GLOBAL = 0.85
+folder = 'data'
+dom_files = [f for f in os.listdir(folder) if 'dom' in f and 'chrome' in f]
```

Comment:
Also add a `"firefox" in f`, to avoid the assumption that Firefox is always the first in the crawler.

Reply:
So I will take intersection of dom_files_chrome and dom_files_firefox